tmva/sofie/src/RModel.cxx

```diff
@@ -318,6 +318,12 @@ namespace SOFIE{
             if (f->second.type == ETensorType::FLOAT){
                fGC += "std::vector<float> ";
             }
+            else if(f->second.type == ETensorType::INT32){
+               fGC += "std::vector<int> ";
```

Comment:
I would  add support here also for INT64 and the UINT32 and UNIT64

Reply:
Done